### PR TITLE
Updated locale for use with WoWAce Translation System

### PR DESCRIPTION
**Pull request by gOOvER**
I used Translations from WoWAce Translation System for enUS. Maybe some strings are missing.

### CODE REVIEW

**Comment by gOOvER** [comment]
i did not delete "locale.lua".

**Comment by Stanzilla** [comment]
So, a few things:

* Don't add the libs to the repo, this will be handled by .pkgmeta going forward
* You need to add AceLocale to .pkgmeta as well.
* We're missing a feature, the print reminder for missing localizations, I would say that is quite handy and worth to keep? So that has to be ported over as well, after that we can remove locale.lua

**Comment by gOOvER** [comment]
> Don't add the libs to the repo, this will be handled by .pkgmeta going forward
I don't know how you handle this outside WoWAce. I know that packager will add it on commit

> You need to add AceLocale to .pkgmeta as well.
Fixed. Missed it :)

**Comment by Stanzilla** [comment]
Yeah in both cases that are open to discussion, the WowAce packager will put the libs in.

File: .pkgmeta [hunk 1]
```diff
@@ -8,6 +8,7 @@ externals:
     SavedInstances/libs/AceGUI-3.0: https://repos.wowace.com/wow/ace3/trunk/AceGUI-3.0
     SavedInstances/libs/AceTimer-3.0: https://repos.wowace.com/wow/ace3/trunk/AceTimer-3.0
     SavedInstances/libs/AceBucket-3.0: https://repos.wowace.com/wow/ace3/trunk/AceBucket-3.0
+	SavedInstances/libs/AceLocale-3.0: https://repos.wowace.com/wow/ace3/trunk/AceLocale-3.0
```
Comment (Stanzilla) [inline .pkgmeta]: indenting :)

**Comment by SacredDuckwhale** [comment]
Isn't this now obsolete? If so, we could just close it :)

**Pull request closed without merging**